packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -127,6 +137,10 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       roapMessage,
       preferTranscoding,
       breakoutsSupported,
+      homerDisclaimerEnabled,
+      isJoining,
+      locale,
+      deviceCapabilities = [],
```

Comment:
There is already a deviceCapabilities definition in the function

Reply:
It's better to allow user to pass deviceCapabilities into SDK, so we don't need to update SDK every time when we want to and a new toggle for deviceCapabilities props.